src/PowerShellEditorServices/Session/PowerShell3Operations.cs

```diff
@@ -6,11 +6,12 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Management.Automation;
 using System.Management.Automation.Runspaces;
 
 namespace Microsoft.PowerShell.EditorServices.Session
 {
+    using System.Management.Automation;
```

Comment:
There's a problem with putting this `using` statement outside the namespace isn't there? We should leave a comment as to why.

Reply:
not sure why yet... :(